### PR TITLE
Constants musc

### PR DESCRIPTION
**Pull request by loganfreeman**
Use ENV to pass variables to translations.

```ruby
link_to t('devise.sessions.institution_users', institution: ENV['institution'])
```

### CODE REVIEW

File: app/views/devise/sessions/new.html.haml [hunk 1]
```diff
@@ -24,7 +24,7 @@
       = t(:devise)[:sessions][:header]
     .form-group.login-buttons
       - if USE_SHIBBOLETH
-        = link_to t(:devise)[:sessions][:institution_users], identity_shibboleth_omniauth_authorize_path(service_request_id: @service_request.try(:id)), class: 'btn btn-primary omniauth-login-button'
+        = link_to t('devise.sessions.institution_users', institution: ENV['institution']), identity_shibboleth_omniauth_authorize_path(service_request_id: @service_request.try(:id)), class: 'btn btn-primary omniauth-login-button'
```
Comment (kyle-glick) [inline app/views/devise/sessions/new.html.haml]: @loganfreeman 
Can you add a default value for this to the `.env.example` file?

Comment (loganfreeman) [reply app/views/devise/sessions/new.html.haml]: added an .env.example:

institution=University.


if the environment variable is missing, the default value is just empty string.

**Comment by Stuart-Johnson** [comment]
@loganfreeman There was an issue with Travis-ci changing up environments, and we had to add things to our travis.yml file. It's pushed up on master, v3.0.0, and v3.0.0b, I think you'll need to update this, and possible the other pull requests, in order for Travis to pass. Thanks.

**Comment by loganfreeman** [comment]
I checked the travis result. 

It said "The command "sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install libqtwebkit-dev libqtwebkit4" failed and exited with 100 during"

I don't understand that.

**Comment by Stuart-Johnson** [comment]
https://www.pivotaltracker.com/story/show/150863897

**Pull request merged**